pkg/skaffold/docker/image.go

```diff
@@ -57,12 +56,16 @@ func RunBuild(ctx context.Context, cli DockerAPIClient, opts *BuildOptions) erro
 		AuthConfigs: authConfigs,
 	}
 
-	buildCtx, err := archive.TarWithOptions(opts.ContextDir, &archive.TarOptions{
-		ChownOpts: &idtools.IDPair{UID: 0, GID: 0},
```

Comment:
@dgageot  Is setting the UID and GID important? Because I don't think I do this anymore. I can add it in the tar file header probably

Reply:
It most likely wont matter for our use case but here is the original issue upstream.

https://github.com/moby/moby/issues/32816

Doesn't matter locally but might matter for CI/CD?